plugins/woocommerce/src/Admin/Features/Blueprint/Init.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare( strict_types = 1);
+
+namespace Automattic\WooCommerce\Admin\Features\Blueprint;
+
+use Automattic\WooCommerce\Admin\PageController;
+
+/**
+ * Class Init
+ *
+ * This class initializes the Blueprint feature for WooCommerce.
+ */
+class Init {
+	/**
+	 * Init constructor.
+	 */
+	public function __construct() {
+		add_action( 'rest_api_init', array( $this, 'init_rest_api' ) );
+		add_filter( 'woocommerce_admin_shared_settings', array( $this, 'add_upload_nonce_to_settings' ) );
+	}
+
+	/**
+	 * Register REST API routes.
+	 *
+	 * @return void
+	 */
+	public function init_rest_api() {
+		( new RestApi() )->register_routes();
```

Comment:
Should we wrap the API registration behind a feature flag?

Reply:
As far as I know, `Init` class won't be loaded unless `blueprint` is enabled. See https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Admin/Features/Features.php#L121

Reply:
Ah, I see. Thanks for the clarification!